browser/mapping_test.go

```diff
@@ -17,6 +17,28 @@ import (
 	k6metrics "go.k6.io/k6/metrics"
 )
 
+// wildcards is a list of extra mappings for our API (api/).
+func wildcards() map[string]string {
+	return map[string]string{
+		"Page.query":             "$",
+		"Page.queryAll":          "$$",
+		"Frame.query":            "$",
+		"Frame.queryAll":         "$$",
+		"ElementHandle.query":    "$",
+		"ElementHandle.queryAll": "$$",
+	}
+}
+
+// getters is a list of mappings that return internal
+// API component properties as Goja objects.
+func getters() map[string]string {
+	return map[string]string{
+		"Page.getKeyboard":    "keyboard",
+		"Page.getMouse":       "mouse",
+		"Page.getTouchscreen": "touchscreen",
+	}
+}
+
```

Comment:
Can we join these two as `customMappings()` or something? I think having two of these is unnecessary (`wildcards()` and `getters()`); it complicates the rest of the code too. A comment in the new `customMappings` function can explain these.

I'll adjust where these are used so you can batch-commit them on Github UI.

```suggestion
// customMappings is a list of custom mappings for our API (api/).
// Some of them are wildcards, such as query to $ mapping; and
// others are for publicly accessible fields, such as mapping
// of page.keyboard to Page.getKeyboard.
func customMappings() map[string]string {
	return map[string]string{
		// wildcards
		"Page.query":             "$",
		"Page.queryAll":          "$$",
		"Frame.query":            "$",
		"Frame.queryAll":         "$$",
		"ElementHandle.query":    "$",
		"ElementHandle.queryAll": "$$",

		// getters
		"Page.getKeyboard":       "keyboard",
		"Page.getMouse":          "mouse",
		"Page.getTouchscreen":    "touchscreen",
	}
}
```

Reply:
Yes, I think it makes complete sense to merge them. Thanks for the suggestion and modifications. Applying them!

Reply:
Seems like GH is not allowing me to batch commit all these suggestions due to:
```
Applying suggestions on deleted lines is currently not supported. Please and try again. 
```
:thinking: 
But I will address them manually and push adding you as co-author.

Reply:
Done in [c05242a](https://github.com/grafana/xk6-browser/pull/751/commits/c05242a2311ae41694ebfba13a59591108b09e4e).